Dockerfile

```diff
@@ -6,8 +6,12 @@ COPY . /usr/src/app
 
 RUN apt-get --yes update \
     && apt-get --yes upgrade \
-    && pip3 install pipenv \
-    && pipenv install --system --deploy --dev
+    && pip3 install pipenv
+
+RUN pip3 install mysqlclient
+RUN pipenv install --system --deploy --dev
 
 EXPOSE 8000
-CMD ["python3", "manage.py", "runserver", "0.0.0.0:8000"]
+
+RUN sleep 10
+CMD ["python3", "/usr/src/app/manage.py", "runserver", "0.0.0.0:8000"]
```

Comment:
The path to `manage.py` should not be hard-coded; it should exist relative to the current working directory, set earlier in the Dockerfile with the `WORKDIR` option. In this case, the change is not necessary.

Reply:
The full path should not be necessary, however I was testing an issue where the django app was not starting on on docker-compose up.  The full path can/should be removed but is not necessary at this moment